use-timescale/integrations/query-admin/qstudio.md

```diff
@@ -0,0 +1,98 @@
+---
+title: qStudio
+excerpt: Connect to your Timescale database with qStudio
+products: [cloud, mst, self_hosted]
+keywords: [connect]
+---
+
+# Connect to Timescale using qStudio
+
+[qStudio][qstudio] is a free SQL GUI to run SQL scripts, browse the tables,
+create charts and export the results. You can install qStudio on every operating
+system and connect to every database including Timescale. You can connect to
+Timescale hosted on your local machine or on a remote server or a Timescale
+service.
+
+For more information about `qStudio`, including download and installation
+instructions, see [timestored.com][qstudio-downloads].
+
+## Before you begin
+
+*   Download and install [qStudio][qstudio-downloads].
+
+<Tabs label="Connect to Timescale with qStudio">
+
+<Tab title="Timescale">
+
+<Procedure>
+
+1.  Sign in to the [Timescale portal][tsc-portal].
+1.  In the `Services` tab, find the service you want to connect to, and check
+    it is marked as `Running`.
+1.  Click the name of the service you want to connect to, and take a note
+     of the `Host`, `Port`, `Database name`, and `Username`.
+1.  [](#)<Optional />If you do not know the password for the service, navigate
+    to the `Operations` tab, and click `Reset password`. You can choose your own
+    password for the service, or allow Timescale to generate a
+    secure password for you. Take a note of your new password.
+1.  Start `qStudio`.
+1.  In the `Server` tab click `Add Server`.
+1.  In the `Server Properties` dialog select `Server Type` as `Postgres`.
+1.  Set `Connect By` as `URL`
+1.  Set the `Host`, `Port`,`Database`, `Username` and `Password`.For more
+    information about these settings,
+    see [the connection section][connection-details].
+1.  Click `Test` to check if the connection is successful.
+1.  Click `Add` to add the database server.
+    The server is listed in the `Server Tree` pane.
+
+    <img class="maincontent__illustration"
+    src="https://s3.amazonaws.com/assets.timescale.com/docs/images/qstudio_timescale.webp"
+    alt="Connecting to Timescale service using qstudio"/>
+
+</Procedure>
+
+</Tab>
+
+<Tab title="Self-hosted Timescale">
+
+<Procedure>
+
+1.  Start `qStudio`.
+1.  In the `Server` tab click `Add Server`.
+1.  In the `Server Properties` dialog select `Server Type` as `Postgres`.
+1.  Set `Connect By`as `Host`.
+1.  Set the `Host`, `Port`,`Database`, `Username` and `Password`. For more
+    information about these settings,
+    see [the connection section][connection-details].
+1.  Click `Test` to check if the connection is successful.
+1.  Click `Add` to add the database server.
+    The server is listed in the `Server Tree` pane.
```

Comment:
Use the bullets for completing the dialog, like the other procedure. ☝️

Reply:
Done!!! Makes sense :)